solidity/contracts/BondedECDSAKeep.sol

```diff
@@ -610,7 +653,19 @@ contract BondedECDSAKeep is IBondedECDSAKeep {
     /// @notice Checks if the keep is currently active.
     /// @dev Throws an error if called when the keep has been already closed.
     modifier onlyWhenActive() {
-        require(isActive, "Keep is not active");
+        require(isActive(), "Keep is not active");
         _;
     }
+
+    /// @notice Gets the owner of the keep.
+    /// @return Address of the keep owner.
+    function getOwner() external view returns (address){
+        return owner;
+    }
+
+    /// @notice Gets the timestamp the keep was opened at.
+    /// @return timestamp the keep was opened at.
+    function getTimestamp() external view returns (uint256){
```

Comment:
The function name doesn't specify which timestamp we want to get, maybe we should name it `getKeepCreationTimestamp` or `getKeepOpenedTimestamp`?

Reply:
Renamed to `getOpenedTimestamp()` since this is a function executed on a keep; same case as with `getOwner()`. I'll rename factory's `getCreationTime` to `getKeepOpenedTimestamp()`